<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecretNew</class>
 <widget class="QWidget" name="SecretNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scan QR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Open image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter manually</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="new_secret_methods">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="qr">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_9"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="image">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_8"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="manual">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="newSecretName"/>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Secret</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="newSecretValue"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::EditCopy"/>
               </property>
               <property name="popupMode">
                <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="cancelAdd">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="confirmAdd">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
